<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>applylesson</class>
 <widget class="QWidget" name="applylesson">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>406</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>542</width>
    <height>406</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>542</width>
    <height>406</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(235, 244, 234);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>34</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请选择开课时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>98</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>节次一：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="week1">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>94</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>星期</string>
   </property>
  </widget>
  <widget class="QComboBox" name="day1">
   <property name="geometry">
    <rect>
     <x>281</x>
     <y>94</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>节次</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>156</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>节次二：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="week2">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>152</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>星期</string>
   </property>
  </widget>
  <widget class="QComboBox" name="day2">
   <property name="geometry">
    <rect>
     <x>281</x>
     <y>152</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>节次</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>214</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>节次三：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="week3">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>210</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>星期</string>
   </property>
  </widget>
  <widget class="QComboBox" name="day3">
   <property name="geometry">
    <rect>
     <x>281</x>
     <y>210</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>节次</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>321</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;楷体&quot;;
color: rgb(17, 99, 56);</string>
   </property>
   <property name="text">
    <string>温馨提示：同一节课一周内不能超过三节！</string>
   </property>
  </widget>
  <widget class="QPushButton" name="subBtn">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>265</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>282</x>
     <y>265</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
